v2/tools/generator/internal/codegen/pipeline/delete_generated_code.go

```diff
@@ -53,6 +53,12 @@ func deleteGeneratedCodeFromFolder(ctx context.Context, outputFolder string) err
 		return err
 	}
 
+	structurePattern := filepath.Join(outputFolder, "**", "*", "structure.txt")
```

Comment:
Totally fine with this as is but we could also just do any `.txt` files, given there aren't many and we have the check for the codegenerated suffix anyway.

Reply:
Agreed.